KeychainExample/App.js

```diff
@@ -13,7 +13,7 @@ import SegmentedControlTab from 'react-native-segmented-control-tab';
 import * as Keychain from 'react-native-keychain';
 
 const ACCESS_CONTROL_OPTIONS = ['None', 'Passcode', 'Password'];
-const ACCESS_CONTROL_OPTIONS_ANDROID = ['Default'];
```

Comment:
I guess we want to have both `None` and `Fingerprint` here?

Reply:
That option is added on the fly for devices that support it.

https://github.com/oblador/react-native-keychain/blob/2d9ade4097bc8aba63d8f67c75e95456140f4e7f/KeychainExample/App.js#L189